contributors/design-proposals/standardization-of-development-process.md

```diff
@@ -0,0 +1,282 @@
+# Proposal to Standardize Development Process
+
+## Links
+```
+---
+Links:
+- Issues:
+  - [someIssueURL]()
+- PRs:
+  - https://github.com/kubernetes/community/pull/967
+- MailingListDiscussions:
+  - https://groups.google.com/forum/#!topic/kubernetes-dev/65A-3ULYPB0
+- Documentation:
+  - [someDocsLinkURL]()
+```
+
+## Responsible SIG(s)
+
+- SIG Release
+- SIG PM
+- SIG Architecture
+- SIG Governance
+- Technical Steering Committee
+
+## Summary
+
+A standardized development process for Kubernetes is proposed in order to:
+- provide a common structure for proposing changes to Kubernetes
+- ensure that the motivation for a change is clear
+- allow for the enumeration stability milestones and stability graduation
+  criteria
+- persist project information in a Version Control System (VCS) for future
+  Kubernauts
+- support the creation of _high value user facing_ information such as:
+  - release notes
+  - release announcement blog
+  - a cross project development road map
+- support development across multiple repositories beyond `kubernetes/kubernetes`
+- begin to move away from using GitHub issues for feature requests and reserve them
+  for tracking work in flight
+
+## Motivation
+
+Today it is becoming fairly difficult to track changes being made across all
+Special Interest Groups (SIGs) within Kubernetes. For cross project SIGs such
+as SIG PM and SIG Release an abstraction beyond a single GitHub Issue or Pull
+request seems to be required in order to understand and communicate upcoming
+changes to Kubernetes. Particularly the generation of release notes and the
+release announcement blog post are rather difficult and have sometimes delayed
+a release due to incompleteness. In a blog post describing the [road to Go 2][],
+Russ Cox explains
+
+> that it is difficult but essential to describe the significance of a problem
+> in a way that someone working in a different environment can understand
+
+unfortunately I believe that our current mechanism for proposing changes by
+opening a GitHub issue against `kubernetes/features` falls short of providing
+an effective vehicle for agreeing on the motivation and the design of a proposed
+change.
+
```

Comment:
I would go one step further and say the current process is insufficient for the scale of the project.  This is a serious risk that needs attention before 1.9 ships.

Reply:
This is great context to put in this PR, but not in this document.  2 years from now "our current mechanism" will mean something different.